core/src/apps/solana/spl/spl_tokens.py

```
@@ -0,0 +1,24 @@
+from typing import Iterator
+
+
+class SPLToken:
+    def __init__(self, symbol: str, mint: str, decimals: int):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)